service/Payment.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen PrestaShop plugin
+ *
+ * Copyright (c) 2020 Adyen B.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ */
+
+namespace Adyen\PrestaShop\service;
+
+class Payment
+{
+    const AFTERPAY_PAYMENT_METHOD = 'afterpay';
```

Comment:
afterpay has multiple products can't we regex on afterpay ? (now missing afterpay_default and afterpay_default_nl and afterpay_touch.

Reply:
I will add them as well into the list. Do we have a precise list of payment method codes which we want to support?